ddl/models.py

```diff
@@ -47,20 +55,43 @@ class Meta:
 
   name = models.TextField()
   menu = models.ForeignKey(Menu, blank=True)
+  address = models.ForeignKey(Address)
+  delivery_fee = models.IntegerField()
 
 class OrderCart(models.Model):
   class Meta:
     db_table = 'order_cart'
 
+  #future fields: promo code, etc
   order_items = models.ManyToManyField(OrderItem, blank=True)
 
+
+  #Add food items to the order cart
+  #sketch???
+  def update(self, order_item_list):
+    self.order_items.append(order_item_list)
+    self.save()
+
+  #Create a new ordercart
+  @classmethod
+  def create(cls, order_items):
+    order_cart = cls(order_items=order_items)
+    order_cart.save()
+    return order_cart
+
 class Order(models.Model):
   class Meta:
     db_table = 'order_table'
 
   customer = models.ForeignKey(Customer)
+  restaurant = models.ForeignKey(Restaurant)
   order_cart = models.ForeignKey(OrderCart)
-  #add delivery once added to table
+  address = models.ForeignKey(Address)
+  has_checked_out = models.BooleanField()
 
+  def update_address(self, address):
```

Comment:
this method seems unnecessary at the current stage, but if so, I would pull out of model as well